javapoet/src/main/java/com/palantir/javapoet/ParameterSpec.java

```diff
@@ -205,8 +205,7 @@ public Builder addAnnotation(Class<?> annotation) {
         }
 
         public Builder addModifiers(Modifier... modifiers) {
-            Collections.addAll(this.modifiers, modifiers);
-            return this;
+            return addModifiers(Arrays.asList(modifiers));
```

Comment:
Nice, this is a sneaky one

Reply:
Side note: The `FieldSpec` and `TypeSpec` builder currently have a varargs variant only (and no `Iterable` one), I am a bit afraid the same issue could re-appear there again in case an `Iterable` overload is added as well.

Reply:
Would you like to add the `Iterable` overload for symmetry in this change?